core/new-gui/src/app/workspace/service/dynamic-schema/schema-propagation/schema-propagation.service.ts

```diff
@@ -60,6 +62,20 @@ export class SchemaPropagationService {
         filter(response => response.code === 0)
       )
       .subscribe(response => {
+        // Find out which input schema changed
+        // console.log("response.result", response.result)
+        for (const operatorID in response.result) {
+          if (isEqual(this.operatorInputSchemaMap[operatorID], response.result[operatorID])) continue;
```

Comment:
I prefer to add a curly brackets even for single line statements. please do
```
if ... {
  continue
}
```

The same comment applies for other places